document/core/exec/runtime.rst

```diff
@@ -47,7 +47,7 @@ Any of the aformentioned references can furthermore be wrapped up as an *externa
    \production{vector} & \vecc &::=&
      \V128.\CONST~\i128 \\
    \production{reference} & \reff &::=&
-     \REFNULL~t \\&&|&
+     \REFNULL~t & (\iff t \neq \typeidx) \\&&|&
```

Comment:
This is correct in spirit, but technically, since meta variables are existentially quantified, a negation like this doesn't do the right thing – you can always take a different type index to make it true. I think you need to say some like "t = absheaptype ∨ t = deftype". Or perhaps spell it out as `\REFNULL~\absheaptype~|~\REFNULL~\deftype`.

Reply:
Thank you for the comment! I fixed this.